pos_default_quantity/static/src/js/pos.js

```diff
@@ -0,0 +1,41 @@
+/*
+  Copyright 2019 Coop IT Easy SCRLfs
+    Robin Keunen <robin@coopiteasy.be>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+*/
+
+odoo.define(
+    'pos_default_quantity.pos_default_quantity',
+
+    function (require) {
+        "use strict";
+
+        var models = require('point_of_sale.models');
+
+        models.load_models({
+            model: 'product.uom.categ',
```

Comment:
using a default_pos_uom, you'd override the loading of product.product to get that field